fv3core/pace/fv3core/stencils/fxadv.py

```diff
@@ -7,8 +7,6 @@
 from pace.util.grid import GridData
 
 
-# TODO: the mix of local and global regions is strange here
-# it's a workaround to specify DON'T do this calculation if on the tile edge
```

Comment:
I removed the comment because there is no mix of local and global regions in the function. It seems to be an outdated comment.

Reply:
there is tough: `with horizontal(region[local_is - 1 : local_ie + 3, :]):` refers to local bounds whereas
```
with horizontal(
            region[:, j_start - 1 : j_start + 1], region[:, j_end : j_end + 2]
        ):
```
is a global region

Reply:
Ah I somehow got lost and I think read the code for `uc_contra_y_edge` instead.

Even with the use of both here, I do want to delete the above comment. It doesn't add any context or information (what's "strange" about it? as far as I can tell this is equally strange to any other region usage). The "workaround" description doesn't say what is being worked around, and only describes the non-"local" regions. It feels like you could attach a comment like this every time we use regions.

Reply:
Under normal conditions we reduce the compute-domain to cover what we want instead of doing a local restriction.
Because this does not work with how regions are implemented - them not extending the computation - we need to work around it by specifying a larger domain to compute on.

So this is the workaround that is talked about here. I can write something and pr it into this branch

Reply:
Right, the local regions are needed so that we can merge stencils despite uc_contra and vc_contra having different output domains. But this is always the normal use case for local regions. We don't put a comment explaining what local regions are every place we use them - comments are not free and easily go out of date. I wouldn't put a `# iterate over the compute domain of the stencil` comment above every `with computation` block, for good reason.

The normal region is unrelated to the uc/vc_contra output domain thing, so I didn't get why there's a comment talking about the mix of region types.

If you think there is a use case for a comment here that goes past repeating what the code already says (e.g. something like we could remove these regions by overcomputing and enabling selective validation, if that's the case), I'm good with that.